### PR TITLE
v3.0.0 - Release Candidate

### PR DESCRIPTION
**Pull request by cayetanobv**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
  This script initializes the plugin, making it known to QGIS.
 """
 
+<<<<<<< HEAD
```
Comment (cvvergara) [inline __init__.py]: this was left over

File: __init__.py [hunk 2]
```diff
@@ -31,6 +33,7 @@ def icon():
     return "icon.png"
 def qgisMinimumVersion():
     return "1.7"
+>>>>>>> master
```
Comment (cvvergara) [inline __init__.py]: also this

Comment (cayetanobv) [reply __init__.py]: ufff I don't know what happened. There was a conflict that I fixed in this file. I've fixed again.

**Comment by cvvergara** [comment]
@cayetanobv 
there is still another HEAD
https://github.com/pgRouting/pgRoutingLayer/blob/490339568d42e14e481d4df809b4c2408d10ae88/pgRoutingLayer.py#L46

**Comment by cayetanobv** [comment]
> @cayetanobv
> there is still another HEAD
> [pgRoutingLayer/pgRoutingLayer.py](https://github.com/pgRouting/pgRoutingLayer/blob/490339568d42e14e481d4df809b4c2408d10ae88/pgRoutingLayer.py#L46)
> 
> Line 46 in [4903395](/pgRouting/pgRoutingLayer/commit/490339568d42e14e481d4df809b4c2408d10ae88)
> 
>  <<<<<<< HEAD

**Comment by cayetanobv** [comment]
@cvvergara I definitely don't know what happened yesterday with those two files. I though I'd fixed but reality is I didn't upload it :(

**Pull request merged**